ansible/pbTestScripts/vagrantPlaybookCheck.sh

```diff
@@ -283,6 +283,10 @@ startVMPlaybookWin()
         # See https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1287#issuecomment-625142917
 	BUILD_ID=dontKillMe vagrant up
 	
+	# Rearm the evaluation license and reboot system
```

Comment:
Could you please give a summary here why this is necessary, why you don't do it in the Vagrantfile or playbook, and how long a it should stay valid afterwards.

Reply:
I've added a link to the referenced issue :-)

Reply:
The only drawback of the issue link: You need to read a lot.